src/bo4e/enum/waehrungseinheit.py

```diff
@@ -0,0 +1,11 @@
+# pylint: disable=missing-module-docstring
+from bo4e.enum.strenum import StrEnum
+
+
+class Waehrungseinheit(StrEnum):
+    """
+    In diesem Enum werden die Währungen und ihre Untereinheiten definiert, beispielsweise für die Verwendung in Preisen.
+    """
+
+    EUR = "EUR"  #: Euro
+    CT = "CT"  #: Eurocent
```

Comment:
I think it's fine cause we are not in other countries with another currency. 
But it seems strange to me, that we have enum `Waehrungscode` but then only `Waehrungseinheit` with `EUR` and `CT`  ^^

Reply:
ja, das problem ist bekannt: #126 
Ich bin damit nur noch nicht durchgedrungen. das v.a. dann probleme wenn der währungscode auf eine währungseinheit trifft. die kannst du nie sauber miteinander verrechnen. betrifft z.B. die Rechnung